app.js

```diff
@@ -0,0 +1,21 @@
+const express = require("express"); //import express module
+const app = express();
+const dotenv = require("dotenv").config(); //import dotenv module
+const mongoose = require("mongoose"); //import mongoose module
+const port = process.env.port;
+
+mongoose.connect(process.env.db_connection);
```

Comment:
I think we should add:
  useNewUrlParser: true // Prevents deprecation errors
  useUnifiedTopology: true// Ensures better connection management and server monitoring

The code should look like this:
mongoose.connect(process.env.DB_CONNECTION, {
  useNewUrlParser: true,
  useUnifiedTopology: true,
});